content/en/docs/ambient/architecture/data-plane/index.md

```diff
@@ -0,0 +1,104 @@
+
+Note that in the case of a request to a `Service`, if the service *has* a waypoint, the request will be sent to its waypoint to enforce L7 policies to the traffic.
+Similarly, in the case of a request to a `Pod` IP, if the pod *has* a waypoint, the request will be sent to its waypoint to enforce L7 policies to the traffic.
+Since it is possible to vary the labels associated with pods in a `Deployment` it is technically possible for
+some pods to use a waypoint while others do not. Users are generally recommended to avoid this advanced use-case.
+
+### Inbound
+
+When a pod in an ambient mesh receives an inbound request, it will be transparently redirected to ztunnel.
+When ztunnel receives the request, it will apply Authorization Policies and forward the request only if the request meets the policies.
+
+A pod can receive HBONE traffic or plaintext traffic.
+By default, both will be accepted by ztunnel.
+Because requests from sources out of mesh will have no peer identity when Authorization Policies are evaluated,
+a user can set a policy requiring an identity (either *any* identity, or a specific one) to block all plaintext traffic.
+
+When the destination is waypoint enabled, if the source is in ambient mesh, the source's ztunnel ensures the request **must** go through
+the waypoint where policy is enforced. However, a workload outside of the mesh doesn't know anything about waypoint proxies therefore it sends
+requests directly to the destination without going through any waypoint proxy even if the destination is waypoint enabled.
+Currently, traffic from sidecars and gateways won't go through any waypoint proxy either and they will be made aware of waypoint proxies
+in a future release.
+
+#### Dataplane details
+
+The L4 ambient dataplane between is depicted in the following figure.
+
+{{< image width="100%"
+link="ztunnel-datapath-1.png"
+caption="Basic ztunnel L4-only datapath"
+>}}
+
+The figure depicts ambient pod workloads running on two nodes W1 and W2 of a Kubernetes cluster. There is a single instance of the ztunnel proxy on each node. In this scenario, application client pods C1, C2 and C3 need to access a service provided by pod S1 and there is no requirement for advanced L7 features such as L7 traffic routing or L7 traffic management so no Waypoint proxy is needed.
+
+The figure shows that pods C1 and C2 running on node W1 connect with pod S1 running on node W2 and their TCP traffic is tunneled through {{< gloss >}}HBONE{{< /gloss >}} tunnel instances that have been created by the ztunnel proxy pods of each node. Mutual TLS (mTLS) is used for encryption as well as mutual authentication of traffic being tunneled. SPIFFE identities are used to identify the workloads on each side of the connection. For more details on the tunneling protocol and traffic redirection mechanism, refer to the guides on [HBONE](/docs/ambient/architecture/hbone) and [ztunnel traffic redirection](/docs/ambient/architecture/traffic-redirection).
```

Comment:
"... through HBONE tunnel instances..." is a bit ambiguous, if you mean the tunnels themselves then I would say

"... through HBONE tunnels..."

Reply:
Yeah I copied this wholesale from the other doc and didn't do a content pass. Will fix.